cmd/collector/collector.go

```diff
@@ -202,6 +239,38 @@ func newCollectorCommand() *cobra.Command {
 	return cmd
 }
 
+func getFlowRecords() []string {
```

Comment:
I think this method just hurts readability (it's a basic accessor, and the caller has to acquire a low anyway). I'd suggest removing it. If the method was making a copy of the slice, it would be a different story.

Reply:
Removed.